### ISSUE TITLE
Zeros not being added as attribute to nodes when using `dataframe_to_tree` in 0.17.0+

### ISSUE THREAD

**Issue by iiwolf** [kayjan/bigtree#224]
**Describe the issue**
When upgrading from `bigtree` version 0.16.2 to 0.17.0, using the `dataframe_to_tree` function where certain values in the DataFrame are 0 results in nodes that don't have that attribute. In other words, if a value is zero in the DataFrame, it doesn't show up at all as an attribute in the node.

I see this note in the changelog:

> Tree Constructor: dataframe_to_tree ignore existing name column, ignore non-attribute columns,
ignore null attribute columns.

So it seems to me 0 is being interpreted as `null`? If this is the desired behavior, is there a way around it?

**Environment**
- Platform: Windows 11
- Python version: 3.10.13
- `bigtree` version: 0.17.0

**To Reproduce**
Here's an example that reproduces the issue:

```python
import pandas as pd
from bigtree import dataframe_to_tree

# Create a DataFrame with some zero values
df = pd.DataFrame({
    'full_path': ['a', 'a/b', 'a/c'],
    'value': [1, 0, 2]
})

# Convert the DataFrame to a tree
tree = dataframe_to_tree(df)

assert hasattr(tree, 'value')
assert not hasattr(tree.children[0], 'value')
assert hasattr(tree.children[1], 'value')
```

In this example, the second node (corresponding to the second row of the DataFrame) should have an attribute 'value' with a value of 0. However, the attribute 'value' is missing from the node.

**Expected behaviour**
I expected all values in the DataFrame, including zeros, to be included as attributes in the nodes of the tree. Each node should have an attribute for each row in the DataFrame, and the value of the attribute should be the value in the 'value' column of the corresponding row.

**Screenshots**
![image](https://github.com/kayjan/bigtree/assets/13955154/cafd294b-4a18-4000-b2d7-4e19c270e938)

**Additional context**
This issue only occurs when using `bigtree` version 0.17.0. When using version 0.16.2, zero values in the DataFrame are correctly included as attributes in the nodes.

Also, thanks for your hardwork on this project! I use it heavily at work!

**Comment by kayjan** [comment]
Hi, thanks for raising this issue with so much details! Happy to hear that you use it heavily for work 😄 

You are right, zero should not have been detected as null and should still be added as a node attribute. The fix is implemented in v0.17.2, do upgrade bigtree with the command `pip install --upgrade bigtree`.

**Comment by iiwolf** [comment]
Kayjan, I've neglected to comment till now. But thank you so much for the quick response! I pulled this down immediately.
